docs/Changelog.md

```diff
@@ -8224,6 +8224,173 @@ This version of the operator has been available since version 8 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Scan-8"></a>**Scan-8**</a>
+
+  Scan can be used to iterate over (specified axes of) one or more scan_input tensors,
+  constructing zero or more scan_output tensors. It combines ideas from general recurrences,
+  functional programming constructs such as scan, fold, map, and zip and is intended to enable
+  generalizations of RNN-like constructs for sequence-to-sequence processing.
+  Other tensors (referred to as state_variables here) can be used to carry a state
+  when iterating from one element to another (similar to hidden-state in RNNs, also referred
+  to as loop-carried dependences in the context of loops). All these tensors are required to
+  have the same shape in each iteration of the loop (a restriction imposed to enable efficient
+  memory allocation). Many common usages involve a single scan_input tensor (where functionality
+  similar to scan, fold and map can be obtained). When more than one scan_input is used,
+  a behavior similar to zip is obtained.
+  
+  The attribute body must be a graph, specifying the computation to be performed in
+  every iteration. It takes as input the current values of the state_variables and
+  the current iterated element of the scan_inputs. It must return the (updated) values
+  of the state_variables and zero or more scan_output_element tensors. The values of the
```

Comment:
Is the number of output tensors constant across loop iterations, so the body potentially updates none of them rather than returns zero scan_output_element tensors?

Reply:
Sorry, I don't understand the question. Yes, the number of outputs produced by the body sub-graph is constant across loop iterations. The values of some of these outputs may not vary from loop-iteration to loop-iteration. I didn't understand the point about zero scan_output_element.

Reply:
If there are zero output elements, what is the purpose of the iterations? Don't they need to produce something, or is it valid to just update the state variables?

Reply:
We will still return the final-value of state-variables (this will be like the output Y_h of RNN). The scan_output_element tensors are concatenated and returned and that will be like the output Y of RNN.